vgc/autopilot_test.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
This file should NOT be here! Test files need to be in `tests/` dir

Reply:
Done